.github/workflows/ci.yml

```
@@ -87,6 +88,195 @@ jobs:
         if: startsWith(matrix.rust, 'nightly')
       - run: cargo minimal-versions build --workspace --all-features --ignore-private
 
+  llvm-cov-nextest-partitioned:
+    name: 1/2 Run tests and coverage with partitioning
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        partition: [1, 2]
+    env:
+      CARGO_TARGET_DIR: target
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          override: true
+          components: llvm-tools-preview
```

actions-rs/* are unmaintained and broken. Please use rustup as existing jobs do.

```suggestion
      - name: Install Rust
        run: rustup toolchain add nightly --no-self-update --component llvm-tools-preview && rustup default nightly
```

(In my project, CI is configured not to start if such an action is used in the workflow.)

---

Thanks, applied. Do you have a link how you've done this? Also it would be good to know what's broken if you have it handy.